test/github-issues/7662/issue-7662.ts

```diff
@@ -0,0 +1,113 @@
+import { expect } from "chai";
+import { Connection } from "../../../src/connection/Connection";
+import {
+    createTestingConnections,
+    closeTestingConnections,
+} from "../../utils/test-utils";
+import { MemoryLogger } from "./memory-logger";
+
+describe("github issues > #7662 postgres extensions installation should be optional", function () {
+    it("should NOT install extensions if option is disabled", async function () {
+        let connection: Connection | null = null;
+        try {
+            const connections = await createTestingConnections({
+                entities: [`${__dirname}/entity/*{.js,.ts}`],
+                enabledDrivers: ["postgres"],
+                schemaCreate: false,
+                dropSchema: true,
+                createLogger: () => new MemoryLogger(true),
+                driverSpecific: {
+                    installExtensions: false,
+                },
+            });
+
+            if (connections.length < 1) {
+                this.skip();
```

Comment:
You should probably `return` so you don't run the rest of the code.

I think that `this.skip()` emits an error somehow but it'd be clearer if you `return this.skip()` or `this.skip();` followed by `return;`